pybrake/test_tdigest.py

```diff
@@ -8,11 +8,8 @@ def test_tdigest_from_bytes():
     _test_digest(digest)
 
 
-def test_tdigest_benchmark_from_bytes(benchmark):
-    def fn():
```

Comment:
What's the problem here? Just curious.

Reply:
Because `benchmark` is not define. It gives below error.

```
file /Users/kaushalprajapati/workspace_py/pybrake/pybrake/test_tdigest.py, line 12
  def test_tdigest_benchmark_from_bytes(benchmark):
E       fixture 'benchmark' not found
>       available fixtures: cache, capfd, capfdbinary, caplog, capsys, capsysbinary, cov, doctest_namespace, monkeypatch, no_cover, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory
>       use 'pytest --fixtures [testpath]' for help on them.
```